src/screens/sync/Devices.tsx

```diff
@@ -136,66 +141,81 @@ const deviceListStyles = StyleSheet.create({
             </Text>
+            <Spacer direction="horizontal" size={spacing.medium} />
+            <TouchableOpacity
+              onPress={() => {
+                setAndOpenModal({
+                  title: m.localDevices.defaultMessage,
+                  description: m.localDeviceDescription.defaultMessage,
+                });
+              }}
+              style={deviceListStyles.infoButton}
+            >
+              <MaterialIcon name="help" size={14} color={colors.DARK_GRAY} />
+            </TouchableOpacity>
           </View>
-        </Pressable>
-      </View>
-      <View style={deviceListStyles.listHeaderContainer}>
-        <Text size="small" color={colors.DARK_GRAY}>
-          {t(m.deviceName)}
-        </Text>
-        <Text size="small" color={colors.DARK_GRAY}>
-          {t(m.lastSynced)}
-        </Text>
+          <Pressable
+            onPress={() => setShouldStart(true)}
+            android_ripple={{ radius: 100 }}
+            style={deviceListStyles.syncButton}
+          >
+            <View style={deviceListStyles.syncButtonContentContainer}>
+              <MaterialIcon
+                name="lightning-bolt"
+                size={20}
+                color={colors.WHITE}
+              />
+              <Spacer direction="horizontal" size={spacing.medium} />
+              <Text size="medium" color={colors.WHITE} bold>
+                {t(m.sync)}
+              </Text>
+            </View>
+          </Pressable>
+        </View>
+        <View style={deviceListStyles.listHeaderContainer}>
+          <Text size="small" color={colors.DARK_GRAY}>
+            {t(m.deviceName)}
+          </Text>
+          <Text size="small" color={colors.DARK_GRAY}>
+            {t(m.lastSynced)}
+          </Text>
+        </View>
+        {[1].map((val) => (
+          <ProgressBar
+            key={val}
+            deviceId={val.toString()}
+            deviceType={val % 2 === 0 ? "desktop" : "mobile"}
+            deviceName={"Example"}
+            date="Feb 12, 2023"
+            syncGroup="local"
+            shouldStart={shouldStart}
+            style={{ marginTop: 10 }}
+          />
+        ))}
       </View>
-      {[1, 3, 2, 5, 262, 26, 22, 34, 27, 235, 98, 45].map((val) => (
-        <ProgressBar
-          key={val}
-          deviceId={val.toString()}
-          deviceType={val % 2 === 0 ? "desktop" : "mobile"}
-          deviceName={"Example"}
-          date="Feb 12, 2023"
-          syncGroup="local"
-          shouldStart={shouldStart}
-          style={{ marginTop: 10 }}
-        />
-      ))}
-    </View>
+    </React.Fragment>
   );
 };
 
-export const Devices = ({ mode }: { mode: ViewMode }) => {
+export const Devices = ({ children }: { children: React.ReactNode }) => {
   const { formatMessage: t } = useIntl();
-  const [status, setStatus] = React.useState<"loading" | "idle">("loading");
+  const [status, setStatus] = React.useState<"loading" | "idle">("idle");
```

Comment:
guessing this was just for debugging?

Reply:
https://github.com/digidem/mapeo-mobile-shell/pull/28/commits/f63808067cd2927413d7e868415aa0a2fcafec8c

Reply:
yeah, due to the `useFocusEffect` every hot reload caused the loading state to reset, which was getting a little tedious!

Reply:
might be worth taking that out actually, now that we have the navigation in place which would reset this state